src/components/Holdings.jsx

```diff
@@ -102,7 +102,7 @@ export default function Holdings() {
           temp.value = formatToDollars(x["value"]);
           return temp;
         })}
-        title="Wallet"
+        title="Assets"
```

Comment:
Could you change this back? I assigned this to myself and fixed it in the spreadsheet in a different way.